app/src/main/res/values/styles_v2.xml

```diff
@@ -84,6 +84,12 @@
     <item name="android:fontFamily">sans-serif-medium</item>
   </style>
 
+  <style name="Clinic.V2.TextAppearance.Black">
```

Comment:
We already have a text style for this. If you look at the Figma file, they are using `h6` with `grey0` text color. We have `Clinic.V2.TextAppearance.H6.Grey0`